src/da.rs

```diff
@@ -74,6 +85,7 @@ impl CelestiaConnection {
         // TODO: Should buffer size be configurable? Is 5 a reasonable default?
         let (tx, rx) = mpsc::channel(5);
 
+        // TODO: Should we handle the error here?
```

Comment:
Yes, if DA layer cannot be initialized the program should exit with a readable error message instead of panicking. This will happen relatively often either from connectivity issues or bad configs